Makefile.toml

```diff
@@ -12,17 +16,17 @@ args = ["clippy", "--all-targets", "--workspace", "--", "-Dwarnings"]
 
 [tasks.ci-build]
 command = "cargo"
-args = ["build", "--all-targets", "--workspace"]
+args = ["build", "--release",  "--all-targets", "--workspace"]
 
 [tasks.ci-test]
 command = "cargo"
-args = ["test", "--lib", "--bins", "--workspace"]
+args = ["test", "--release", "--lib", "--bins", "--workspace"]
 
 [tasks.ci-doc-test]
 command = "cargo"
-args = ["test", "--doc", "--workspace"]
+args = ["test", "--release", "--doc", "--workspace"]
 
 [tasks.ci-docs]
 env = { "RUSTDOCFLAGS" = "-Dwarnings" }
 command = "cargo"
-args = ["doc", "--no-deps", "--document-private-items", "--workspace"]
\ No newline at end of file
+args = ["doc", "--release", "--no-deps", "--document-private-items", "--workspace"]
```

Comment:
suggestion: what do you think about pulling these two tasks up before `ci-test`? I think they are both much faster than testing and when I run CI locally, it's nice to not have to sit through the tests to figure out if my doc built correctly.

I guess doc-test could optionally stay after regular tests.

Reply:
I agree with this.